### ISSUE TITLE
In Safari, doc.height() is always 1 pixel more than last_scroll_height

### ISSUE THREAD

**Issue by jsit** [leafo/sticky-kit#216]
No idea why this is happening, but it results in the `.is_stuck` class being removed and re-added on every little scroll event. I changed line 119 (within the `tick()` function) to read:

`if (!recalced && Math.abs(doc.height() - last_scroll_height) > 1) {`

And it works.

Safari 10.0.2.

**Comment by carasmo** [comment]
Thank you!

**Comment by TrilipuT** [comment]
yeah, very strange bug... But actually safari also very strange browser )
